.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Rules can be configured as needed. I just threw in what seemed to be existing convention (single quotes, no semi).